bouture.js

```diff
@@ -1,6 +1,7 @@
'th', 'tr'], border: ['img', 'object', 'table'], buffered: ['audio', 'video'], challenge: 'keygen', charset: ['meta', 'script'], checked: ['command', 'input'], cite: ['blockquote', 'del', 'ins', 'q'], class: '*', code: 'applet', codebase: 'applet', color: ['basefont', 'font', 'hr'], cols: 'textarea', colspan: ['td', 'th'], content: 'meta', contenteditable: '*', contextmenu: '*', controls: ['audio', 'video'], coords: 'area', crossorigin: ['audio', 'img', 'link', 'script', 'video'], data: 'object', datetime: ['del', 'ins', 'time'], default: 'track', defer: 'script', dir: '*', dirname: ['input', 'textarea'], disabled: ['button', 'command', 'fieldset', 'input', 'keygen', 'optgroup', 'option', 'select', 'textarea'], download: ['a', 'area'], draggable: '*', dropzone: '*', enctype: 'form', for: ['label', 'output'], form: ['button', 'fieldset', 'input', 'keygen', 'label', 'meter', 'object', 'output', 'progress', 'select', 'textarea'], formaction: ['input', 'button'], headers: ['td', 'th'], height: ['canvas', 'embed', 'iframe', 'img', 'input', 'object', 'video'], hidden: '*', high: 'meter', href: ['a', 'area', 'base', 'link'], hreflang: ['a', 'area', 'link'], 'http-equiv': 'meta', icon: 'command', id: '*', integrity: ['link', 'script'], ismap: 'img', itemprop: '*', keytype: 'keygen', kind: 'track', label: 'track', lang: '*', language: 'script', list: 'input', loop: ['audio', 'bgsound', 'marquee', 'video'], low: 'meter', manifest: 'html', max: ['input', 'meter', 'progress'], maxlength: ['input', 'textarea'], minlength: ['input', 'textarea'], media: ['a', 'area', 'link', 'source', 'style'], method: 'form', min: ['input', 'meter'], multiple: ['input', 'select'], muted: ['audio', 'video'], name: ['button', 'form', 'fieldset', 'iframe', 'input', 'keygen', 'object', 'output', 'select', 'textarea', 'map', 'meta', 'param'], novalidate: 'form', open: 'details', optimum: 'meter', pattern: 'input', ping: ['a', 'area'], placeholder: ['input', 'textarea'], poster: 'video', preload: ['audio', 'video'], radiogroup: 'command', readonly: ['input', 'textarea'], rel: ['a', 'area', 'link'], required: ['input', 'select', 'textarea'], reversed: 'ol', rows: 'textarea', rowspan: ['td', 'th'], sandbox: 'iframe', scope: 'th', scoped: 'style', seamless: 'iframe', selected: 'option', shape: ['a', 'area'], size: ['input', 'select'], sizes: ['link', 'img', 'source'], slot: '*', span: ['col', 'colgroup'], spellcheck: '*', src: ['audio', 'embed', 'iframe', 'img', 'input', 'script', 'source', 'track', 'video'], srcdoc: 'iframe', srclang: 'track', srcset: 'img', start: 'ol', step: 'input', style: '*', summary: 'table', tabindex: '*', target: ['a', 'area', 'base', 'form'], title: '*', translate: '*', type: ['button', 'input', 'command', 'embed', 'object', 'script', 'source', 'style', 'menu'], usemap: ['img', 'input', 'object'], value: ['button', 'option', 'input', 'li', 'meter', 'progress', 'param'], width: ['canvas', 'embed', 'iframe', 'img', 'input', 'object', 'video'], wrap: 'textarea'}
```

Comment:
It might be worth indenting this object for the purpose of being more easily maintained

Reply:
This could also be a Map